scripts/configure-installer-tests-cluster.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Setup to find nessasary data from cluster setup
+## Constants
+HTPASSWD_FILE="./htpass"
+USERPASS="developer"
+HTPASSWD_SECRET="htpasswd-secret"
+# Overrideable information
+AUTH_DIR=${AUTH_DIR:-"$(pwd)/auth"}
+KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
+KUBEADMIN_PASSWORD_FILE_NAME=${KUBEADMIN_PASSWORD_FILE_NAME:-"kubeadmin-password"}
+KUBECONFIG_FILE_NAME=${KUBECONFIG_FILE_NAME:-"kubeconfig"}
+
+# CALCULATED INFORMATION
+KUBEADMIN_PASSWORD_FILE="${AUTH_DIR}/${KUBEADMIN_PASSWORD_FILE_NAME}"
+
+# Exported to current env
+export KUBECONFIG="${AUTH_DIR}/${KUBECONFIG_FILE_NAME}"
+
+# List of users to create
+USERS="developer odonoprojectattemptscreateproject odosingleprojectattemptscreate odologinnoproject odologinsingleproject1"
+
+# Check if nessasary files exist
+if [ ! -f $KUBEADMIN_PASSWORD_FILE ]; then
+    echo "Could not find kubeadmin password file"
+    exit 1
+fi
+
+if [ ! -f $KUBECONFIG ]; then
+    echo "Could not find kubeadm password file"
+    exit 1
+fi
+
+# Get kubeadmin password from file
+KUBEADMIN_PASSWORD=`cat $KUBEADMIN_PASSWORD_FILE`
+
+# Remove existing htpasswd file, if any
+if [ -f $HTPASSWD_FILE ]; then
+    rm -rf $HTPASSWD_FILE
+fi
+
+# Set so first time -c parameter gets applied to htpasswd
+HTPASSWD_CREATED=" -c "
+
+# Create htpasswd entries for all listed users
+for i in `echo $USERS`; do
+    htpasswd -b $HTPASSWD_CREATED $HTPASSWD_FILE $i $USERPASS
+    HTPASSWD_CREATED=""
+done
+
+# Login as admin user
+oc login -u $KUBEADMIN_USER -p $KUBEADMIN_PASSWORD
+
+# Create secret in cluster, removing if it already exists
+oc get secret $HTPASSWD_SECRET
```

Comment:
Unnecessary use of ```oc get secret ...``` i guess.

Reply:
Not really, i am checking if that secret already exist. Makes script usable multiple timeson same cluster, if u add or remove users